### DISCUSSION TITLE
How does Lume update dependencies?

### DISCUSSION THREAD

**Discussion by MidAutumnMoon** [lumeland/lume#657] [Q&A] [answered]
Hi, I'm fairly new to Deno ecosystem, and one thing bothers me is how to update dependencies. 

I saw Lume update deps quite often, how does Lume manage its dependencies? Thanks!

**Comment by oscarotero** [comment, marked as answer]
There's a task to update deps: https://github.com/lumeland/lume/blob/923e36dc7fd3288c6365c2277e9becfdb684015d/deno.json#L15

It uses the nudd library https://github.com/oscarotero/nudd

**Reply by MidAutumnMoon** [reply]
Didn't notice that, thanks!
